src/main/java/org/broadinstitute/consent/http/resources/DataRequestCasesResource.java

```diff
@@ -35,22 +35,6 @@ public DataRequestCasesResource(ElectionService electionService, PendingCaseServ
         this.summaryService = summaryService;
     }
 
-    @GET
-    @Path("/pending")
-    @RolesAllowed({CHAIRPERSON, MEMBER})
-    public Response getDataRequestPendingCasesByAuthUser(@Auth AuthUser authUser) {
-        List<PendingCase> pendingCases = pendingCaseService.describeDataRequestPendingCases(authUser);
-        return Response.ok().entity(pendingCases).build();
-    }
```

Comment:
This is still in use here: https://github.com/DataBiosphere/duos-ui/blob/develop/src/libs/ajax.js#L704